kalite/distributed/tests/features/steps/exercises.py

```diff
@@ -0,0 +1,45 @@
+from behave import *
+from django.conf import settings
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+TIMEOUT = 3
+
+@given("I open an exercise")
```

Comment:
There is something so appealing about this way of labeling steps. Really hope we can build up a good library of steps and encourage non-devs to create Gherkin test specifications with them.